platform.md

```diff
@@ -293,17 +314,26 @@ Usage:
 - **If** `<daemon>` is `true`, `<image>` MUST be either a valid image reference or an imageID
 - The lifecycle MUST accept valid references to non-existent images without error.
 
+##### Outputs
 | Output             | Description
 |--------------------|----------------------------------------------
-| [exit status]      | Success (0), or error (1+)
+| [exit status]      | (see Exit Code table below for values)
 | `/dev/stdout`      | Logs (info)
 | `/dev/stderr`      | Logs (warnings, errors)
 | `<analyzed>`       | Analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
 | `<layers>/<buidpack-id>/<layer>.sha`  | Files containing the diffID of each analyzed layer
 | `<layers>/<buidpack-id>/<layer>.toml` | Files containing the layer content metadata of each analyzed layer (see data format in [Buildpack Interface Specification](buildpack.md))
 
+| Exit Code | Result|
+|-----------|-------|
+| `0`       | Success
+| `11`      | Platform API incompatibility error
+| `12`      | Buildpack API incompatibility error
+| `1-10`, `13-99` | Generic lifecycle errors
+| `200-299` | Analysis-specific lifecycle errors
+
 - The lifecycle MUST write [analysis metadata](#analyzedtoml-toml) to `<analyzed>` if `<image>` is accessible.
-- **If** `<skip-layers>` is `false` the lifecycle MUST analyze any app image layers or cached layers created by any buildpack present in the provided `<group>`.
+- **If** `<skip-layers>` is `false` the lifecycle MUST NOT analyze any app image layers or cached layers created by any buildpack present in the provided `<group>`.
 - **If** `<skip-layers>` is `true` the lifecycle MUST NOT perform layer analysis.
```

Comment:
Maybe it's worth changing it to `if-else`?

Reply:
done!